### ISSUE TITLE
Add some OQS tests to x509/x509_test.cc and evp/evp_test.cc

### ISSUE THREAD

**Issue by xvzcf** [open-quantum-safe/boringssl#60]
This might preempt issues such as https://github.com/open-quantum-safe/boringssl/issues/58

**Comment by claucece** [comment]
I'll start looking at this this week ;)

**Comment by baentsch** [comment]
#68 might cause a significant code base change that could throw anybody's work back, so question to @claucece : Do you have something to merge on this issue that should be merged before adding in #68?

**Comment by claucece** [comment]
@baentsch yes, I was doing intial testing. But, don't feel stopped by it. I'll rebase and fix any conflicts.
